### PR TITLE
Add text to README file

### PR DESCRIPTION
**Pull request by MTCawthray**
I added some text to the README file for your review.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -0,0 +1,2 @@
+# This is a README file!
+## congrats on making it this far
```
Comment (EricSergeant) [inline README.md]: We should capitalize "congrats" in line 2.

**Review by EricSergeant — COMMENTED** [review]
One grammar suggestion

**Comment by MTCawthray** [comment]
I will fix and repush.

**Comment by EricSergeant** [comment]
Good change, merging.

**Pull request merged**